varya/assets/sass/elements/_forms.scss

```diff
@@ -42,8 +42,12 @@ input[type=checkbox] + label {
 	line-height: 1em;
 }
 
+// Search widget styles
```

Comment:
Lets move these search widget specific styles to `varya/assets/sass/components/widgets/_style.scss`